services/ui-src/src/utils/auth/userProvider.test.tsx

```diff
@@ -7,38 +7,35 @@ import { UserContext } from "./userContext";
 import { Auth } from "aws-amplify";
 // utils
 import { RouterWrappedComponent } from "utils/testing/setupJest";
+import { errorHandler } from "utils/errors/errorHandler";
```

Comment:
This file looks like it has changed a lot, and it kinda has. So buckle up. But also it's not so bad.